### ISSUE TITLE
Re-work SDOE paragraph and related figure.

### ISSUE THREAD

**Issue by ksbeattie** [CCSI-Toolset/FOQUS#423]
When generating the docs, I'm seeing the following errors (blocked out in fix-width below):
-----
(foqus) ksb@ubn2-ksb[docs] 00:38:59 (0)$ make html
SPHINX=1 sphinx-build -b html -d build/doctrees  -w sphinx-build.log -T  source build/html
Running Sphinx v1.8.4
making output directory...
loading intersphinx inventory from https://docs.python.org/objects.inv...
intersphinx inventory has moved: https://docs.python.org/objects.inv -> https://docs.python.org/3/objects.inv
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 55 source files that are out of date
updating environment: 55 added, 0 changed, 0 removed
reading sources... [100%] references
```
/home/ksb/Projects/CCSI/github/CCSI-Toolset/FOQUS/docs/source/chapt_sdoe/index.rst:91: WARNING: Error in "figure" directive:
invalid option block.

.. figure:: figs/3_scatterplot_inputs.png
   :alt: SDOE plot of inputs
   SDOE plot of inputs

   [fig:3_scatterplot_inputs]
/home/ksb/Projects/CCSI/github/CCSI-Toolset/FOQUS/docs/source/chapt_sdoe/index.rst:105: WARNING: Error in "figure" directive:
invalid option block.

.. figure:: figs/4_scatterplot_aggregated.png
   :alt: SDOE plot of aggregated inputs
   SDOE plot of aggregated inputs

   [fig:4_scatterplot_aggregated]
/home/ksb/Projects/CCSI/github/CCSI-Toolset/FOQUS/docs/source/chapt_sdoe/index.rst:113: WARNING: Error in "figure" directive:
invalid option block.

.. figure:: figs/5_SDOE_page2.png
   :alt: SDOE second window
   SDOE second window

   [fig:5_SDOE_page2]
```
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] references
generating indices... genindex
writing additional pages... search
copying images... [100%] chapt_uq/tutorial/../figs/tutorial/10a_RosenbrockUA.png
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 3 warnings.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
----
related to #384 and #418

**Comment by ksbeattie** [comment]
I see part of the problem.  In `index.rst` it looks like you need a space after the `::alt::`  As in, rather than:

```
 .. figure:: figs/3_scatterplot_inputs.png
    :alt: SDOE plot of inputs
    SDOE plot of inputs

    [fig:3_scatterplot_inputs]
```

You want:

```
 .. figure:: figs/3_scatterplot_inputs.png
    :alt: SDOE plot of inputs

    SDOE plot of inputs

    [fig:3_scatterplot_inputs]
```

Fixing that in 3 places removes the above errors, but then it looks like the file

    chapt_sdoe/figs/4_scatterplot_aggregated.png

is missing.

**Comment by ksbeattie** [comment]
btw, I've regenerated these docs on on the gh_pages branch, so they can be seen here:

https://ccsi-toolset.github.io/FOQUS/dev/index.html

**Comment by sotorrio1** [comment]
@candcook have you uploaded the missing file "chapt_sdoe/figs/4_scatterplot_aggregated.png" 
@ksbeattie any other errors with this?

**Comment by ksbeattie** [comment]
I believe that's it, though we should look again after @makaylas work in PR #439

**Comment by ksbeattie** [comment]
@makaylas fixed this in #439

**Comment by ksbeattie** [comment]
Re-opening this as the file `chapt_sdoe/figs/4_scatterplot_aggregated.png` is still missing.

@sotorrio1, @candcook anyone have this image?

**Comment by sotorrio1** [comment]
I don't have it, but if @candcook describes it I can generate it.

**Comment by ksbeattie** [comment]
Is it described well enough there in the docs for you to generate it?

**Comment by sotorrio1** [comment]
@ksbeattie Just checked the description in the SDOE documentation and I think that image doesn't exist because it's a placeholder. I think @candcook is talking about a feature she wants in the plotting capability but it's not implemented yet.

**Comment by candcook** [comment]
Yes, the missing plot is one that is forthcoming. When I was doing the writing, I did not know how quickly this version would be included in a release. 
When some of the new functionality is added, I will update all of the figures and include new ones that showoff the capabilities.
I also plan on adding several new examples to illustrate the methods.

**Comment by ksbeattie** [comment]
@candcook, sorry about the miscommunication on the timing of this release.  We had originally scheduled it for Feb, so I'd really like to get this out this week.  Is there anything that you can add in the next day or should we go with that is there now?

fyi: you can see the latest version here: https://foqus.readthedocs.io/en/latest/

**Comment by sotorrio1** [comment]
@candcook should decide this, but I think it could look just fine if for now we just delete the following paragraph:

'''There are options to view the aggregated files for both the candidate and history files, with a similar interface as was shown in step 3. In addition, a single plot of the combined candidate and history files can be viewed, by (ADD DETAILS WHEN AVAILABLE). In this plot the (BLACK) points represent the candidate locations, while the (RED) points represent already collected data from the history file.'''

Since it's the only text associated with the missing screenshot 'SDOE plot of aggregated inputs'.

Thoughts?

**Comment by ksbeattie** [comment]
@sotorrio1, I've gone ahead and commented that paragraph out, so that we can move forward with the Feb release.

I'll leave this issue open for @candcook to updae the paragraph and related figure.

**Comment by ksbeattie** [comment]
@candcook has started working on this again.

**Comment by sotorrio1** [comment]
@candcook has a finished version ready for @atowfiq and myself to review. She's planning on creating a PR by the end of the week.

**Comment by sotorrio1** [comment]
Just approved and merged PR #554 
This should take care of the SDoE documentation for now and I recommend closing this issue
